gulpfile.babel.js

```diff
@@ -154,6 +154,18 @@ gulp.task('html', () => {
       removeStyleLinkTypeAttributes: true,
       removeOptionalTags: true
     })))
+    .pipe($.if('*.html', $.minifyInline({
+            js: {
+                output: {
+                    comments: true
+                }
+            },
+            jsSelector: 'script[type!="text/x-handlebars-template"]',
+            css: {
+                keepSpecialComments: 1
+            },
+            cssSelector: 'style[do-not-minify!="true"]'
```

Comment:
`data-do-not-minify` would be a better attribute name to use. Data attributes exist for custom things like this.